### DISCUSSION TITLE
Draw or reorder over fullscreen?

### DISCUSSION THREAD

**Discussion by gavsiu** [yshui/picom#834] [Q&A] [answered]
I'm using [lead](https://github.com/MageJohn/lead) for hot-corners shortcuts. It says it uses a compositor to make the zones transparent. The problem is in full-screen (Firefox or YouTube), the zones are covered and do not activate.

I'm not sure how picom renders things, but is there a way to reorder [lead](https://github.com/MageJohn/lead) to be above everything?

OS: Arch
WM: bpswm
App: [lead](https://github.com/MageJohn/lead)

**Comment by gavsiu** [comment]
In lead debug mode, the monitor names and hot corner shortcuts are shown and they draw over full-screen apps, but when I turn debug off and everything is hidden/transparent, it doesn't work over full-screen.

**Comment by tryone144** [comment, marked as answer]
I haven't used this tool personally, but I suspect this is rather a problem with your window manager than picom.
Picom (or any compositor for that matter) only does the rendering. It has nothing to do with the actual stacking order of the applications.

There might be a way to force bspwm to keep a specific window (like lead) in the foreground, even over full-screen windows.

**Reply by gavsiu** [reply]
Thanks for the hint on where to look.

`bspc rule -a lead layer=above`
